docs/app/insights.md

```diff
@@ -1,81 +1,199 @@
+> able to setup key data metrics in just a few clicks.
 
-[[toc]]
+![image of dashboard(s)](../assets/insights-images/insights_dashboards.gif)
 
-## Browsing Dashboards
+- [How it Works](#how-it-works)
+- [Dashboards Overview](#dashboards-overview)
+- [Panels Overview](#panels-overview)
+  - [Panel Header Options](#panel-header-options)
+  - [Label Overview](#label-overview)
+  - [List Overview](#list-overview)
+  - [Metrics Overview](#metrics-overview)
+    - [Style and Format Options](#style-and-format-options)
+  - [Time Series Overview](#time-series-overview)
+  - [Advanced Filtering](#advanced-filtering)
+- [Tutorial Video](#tutorial-video)
+- [Extensibility Options](#extensibility-options)
 
-Clicking into the Insights module immediately presents a listing of all dashboards you have access to. These are shown
-in the navigation bar and a listing on the main page.
+## How it Works
 
-## Creating a Dashboard
+![gif/video of setting up dashboard and panels](image.gif)
 
-1. Click the "Create Dashboard" action button (`+`) in the header.
-2. Fill in the dashboard name, icon and note
-3. Click "Save"
+1. Create a new dashboard by clicking the `+` icon in the upper right hand corner.
+2. Name your dashboard and add a brief note to describe its purpose.
+3. Click the edit icon in the upper right hand corner, then hit `+` to add a panel.
+4. Select a panel type, then use Panel Options to setup metrics.
 
-## Editing a Dashboard
+### Intro Video
 
-1. Click on the Insights module to go to the Dashboard listing
-2. Click the right-hand context menu (`...`) for the dashboard you want to edit
-3. Make any edits, and then click "Save"
+This [Intro video](https://www.example.com) will give you a firm overview of the insights module.
 
-::: tip Editing the Dashboard Panels
+## Dashboards Overview
 
-To edit the Panels within the dashboard, please see: [Editing a Panel](#editing-a-panel)
+A Directus dashboard provides a drag-and-drop, snappy-grid area where you create and edit panels, which provide
+analytics information. The Dashboard area automatically expands as you add more and more panels. In theory, a dashboard
+area can expand infinitely large... but in practice, users will probably only want to build dashboards as large as the
+screen they will be viewing on. You are able to create as many dashboards as you need. Additionally, the Dashboard view,
+edit, and create permissions can be turned on and off based on user role.
 
-:::
+![image of Empty Dashboard](../assets/insights-images/insights_dashboard-grid.gif)
 
-## Deleting a Dashboard
+### Create, Edit, and Delete Dashboards
 
-1. Click on the Insights module to go to the Dashboard listing
-2. Click the right-hand context menu (`...`) for the dashboard you want to edit
-3. Choose "Delete Dashboard" and click to confirm
+![image of dashboard with number by add and edit/delete buttons](../assets/insights-images/insights_dashboard_edit-and-delete-buttons.gif)
```

Comment:
Since we may want users to quickly find these images on Google you should improve the description keywords.

For example, check this better approach
```suggestion
![Edit / delete panel in Directus Insights / dashboard](../assets/insights-images/insights_dashboard_edit-and-delete-buttons.gif)
```

Do you see the relevant keywords? Basically you just need to think how you would do a search to find this image. The same applies to all the other images 👍

Reply:
Yes, thanks @joselcvarela